src/Components/Shared/Button/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './button.module.css';
+
+const Button = ({ text, clickAction, type, disabled }) => {
+  let classNameStyle;
+  switch (type) {
```

Comment:
I think it would be easier to change style with a prop and not with a switch of types, you can use the admin button as a reference

Reply:
I can see that there is an argument for that, but by including the switch method you dont have to write 'styles.selectedStyle' as a prop, as you need to do in the case of Admin.

Reply:
Delfi is right, you could do it in a much simpler way and avoid using the switch statement if you write it like this::
`const classNameStyle = styles[type]`